servepkx/python/servepkx.py

```diff
@@ -1,15 +1,31 @@
+import argparse
 import socket
-import sys
-
-def sendpkx(arg): 
-	s = socket.socket()
-	s.connect((sys.argv[1], 9000))
-	f = open (arg, "rb")
-	l = f.read(232)
```

Comment:
is there a reason on why read and send 232 bytes, python socket has a method called send_all to avoid a " while read" loop